### PR TITLE
Implement V525

### LINKED ISSUE #171: Validation (2017): V525 (HOEPA Status)
**Edit #:** V525
**Type:** Validity
**Category:** Loan/Application Register (only)
**Column:** HOEPA Status

HOEPA status must = 1 or 2.

Additional resources:
* [FFIEC's 2016 HMDA File Spec](https://www.ffiec.gov/hmda/pdf/spec2016.pdf)
* [FFIEC's 2016 HMDA Edits](https://www.ffiec.gov/hmda/pdf/edit2016.pdf)
* [CFPB's 2017 HMDA File Spec](http://www.consumerfinance.gov/hmda/static/for-filers/2017/2017-HMDA-File-Specifications.pdf)

### PR DESCRIPTION
**Pull request by kgudel**
Closes #171

### CODE REVIEW

File: validation/src/test/scala/hmda/validation/rules/lar/validity/V525Spec.scala [hunk 1]
```diff
@@ -0,0 +1,26 @@
+package hmda.validation.rules.lar.validity
+
+import hmda.model.fi.lar.{ Loan, LoanApplicationRegister }
+import hmda.validation.dsl.{ Failure, Success }
+import hmda.validation.rules.lar.LarEditCheckSpec
+import org.scalacheck.Gen
+
+class V525Spec extends LarEditCheckSpec with BadValueUtils {
+  property("HOEPA Type must = 1, or 2") {
```
Comment (jmarin) [inline validation/src/test/scala/hmda/validation/rules/lar/validity/V525Spec.scala]: Where does the word `Type` come from? Given that it is a loaded term in Scala for many reasons I hesitate to use it unless there is a good reason. The edits document uses HOEPA `status`

Comment (kgudel) [inline validation/src/test/scala/hmda/validation/rules/lar/validity/V525Spec.scala]: it was arbitrarily chosen, that's good point I hadn't thought of, I'll change it to status.

File: validation/src/test/scala/hmda/validation/rules/lar/validity/V525Spec.scala [hunk 2]
```diff
@@ -0,0 +1,26 @@
+package hmda.validation.rules.lar.validity
+
+import hmda.model.fi.lar.{ Loan, LoanApplicationRegister }
+import hmda.validation.dsl.{ Failure, Success }
+import hmda.validation.rules.lar.LarEditCheckSpec
+import org.scalacheck.Gen
+
+class V525Spec extends LarEditCheckSpec with BadValueUtils {
```
Comment (jmarin) [inline validation/src/test/scala/hmda/validation/rules/lar/validity/V525Spec.scala]: I get a compilation error on your branch, it's not finding `BadValueUtils` and `intOutsideRange`. Can you check that you have merged this functionality into your PR?

Comment (kgudel) [inline validation/src/test/scala/hmda/validation/rules/lar/validity/V525Spec.scala]: the most recent commit should fix this.

**Pull request closed without merging**